contributors/design-proposals/pod-pid-namespace.md

```diff
@@ -0,0 +1,62 @@
+# Shared PID Namespace for the Docker Runtime
+
+Pods share many namespaces, but the ability to share a PID namespace was not
+supported by Docker until version 1.12. This document proposes how to roll out
+support for sharing the PID namespace in the docker runtime.
```

Comment:
Actually, one other point this raises: I think we should have a corollary change in the CRI clarifying that _all new runtime implementers must use a shared PID namespace_. Otherwise even after this change goes through we might have other implementations that have (inadvertently or otherwise) implemented the old/undesirable behaviour.